xchange-coinbasepro/src/main/java/info/bitrich/xchangestream/coinbasepro/CoinbaseProStreamingExchange.java

```diff
@@ -19,6 +19,8 @@
 public class CoinbaseProStreamingExchange extends CoinbaseProExchange implements StreamingExchange {
     private static final String API_URI = "wss://ws-feed.pro.coinbase.com";
     private static final String SANDBOX_API_URI = "wss://ws-feed-public.sandbox.pro.coinbase.com";
+    private static final String PRIME_API_URI = "wss://ws-feed.prime.coinbase.com";
```

Comment:
Has this got mixed up with #476 ?

Reply:
yup

Reply:
fixed in commit d80358f69ab5f37c1d6d5d912e8d2b0ebb23e0d6